lib/engine/db/dbproxy/local_table.ts

```diff
@@ -21,22 +21,22 @@ import * as Tp from 'thingpedia';
 export default class LocalTable<RowType> {
     name : string;
     private _baseUrl : string;
-    private _userId : number;
+    private _auth : string|undefined;
 
-    constructor(name : string, baseUrl : string, userId : number) {
+    constructor(name : string, baseUrl : string, accessToken : string|undefined) {
```

Comment:
should accessToken be required since dbproxy api will be broken without a user id ?

Reply:
I wanted to keep the interface a bit more generic in case we change how dbproxy works in the future. The contract right now is that the platform decides if the access token is required or not. If the access token is provided, it is also required by the API.